x-pack/plugins/spaces/public/share_saved_objects_to_space/components/share_to_space_form.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import './share_to_space_form.scss';
+import React, { Fragment } from 'react';
+import { EuiHorizontalRule, EuiFormRow, EuiCallOut, EuiButton } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { ShareOptions, SpaceTarget } from '../types';
+import { SelectableSpacesControl } from './selectable_spaces_control';
+
+interface Props {
+  spaces: SpaceTarget[];
+  onUpdate: (shareOptions: ShareOptions) => void;
+  shareOptions: ShareOptions;
+  showShareWarning: boolean;
+  makeCopy: () => void;
+}
+
+export const ShareToSpaceForm = (props: Props) => {
+  const setSelectedSpaceIds = (selectedSpaceIds: string[]) =>
+    props.onUpdate({ ...props.shareOptions, selectedSpaceIds });
+
+  const getShareWarning = () => {
+    if (!props.showShareWarning) {
+      return null;
+    }
+
+    return (
+      <Fragment>
+        <EuiCallOut
+          title={
```

Comment:
The visual weight of this callout feels off balance. How do you feel about shrinking this (and the contained button) a bit?

```suggestion
          size="s"
          title={
```